esphome/components/teleinfo/sensor.py

```diff
@@ -0,0 +1,33 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import sensor, uart
+from esphome.const import CONF_ID, CONF_SENSOR, CONF_NAME, ICON_FLASH, UNIT_WATT_HOURS
+
+DEPENDENCIES = ['uart']
+
+teleinfo_ns = cg.esphome_ns.namespace('teleinfo')
+TELEINFO = teleinfo_ns.class_('TeleInfo', cg.PollingComponent, uart.UARTDevice)
```

Comment:
```suggestion
TeleInfo = teleinfo_ns.class_('TeleInfo', cg.PollingComponent, uart.UARTDevice)
```

Class name should match the variable name (as in the rest of the codebase)

Reply:
I have done the modifications.
Thanks for reviewing